### DISCUSSION TITLE
Naming convention for boolean props/hooks

### DISCUSSION THREAD

**Discussion by harrywebdev** [evoluhq/evolu#61] [General]
I understand this is a pretty subjective matter. Anyway, I came across this hook
https://github.com/evoluhq/evolu/blob/15fa758305e9d04d845363d0a86f0cf95b0fde96/packages/evolu/src/db.ts#L89-L94

and I thought it would be nicer if the state is inverted (e.g. `areLoaded` -> `areLoading`). The other suggestion would be to prefix boolean with `is` to imply the type from a quick glance.

```
export const useIsEvoluFirstDataLoading = (): boolean =>
  useSyncExternalStore(
    listen,
    () => readonlyRecord.isEmpty(queriesRowsCacheRef.read()),
    constTrue
  );
```

Btw. I did not try this code, just rewrote it based on an assumption.

**Comment by steida** [comment]
It's probably better, but it's temporal code I will deprecate once React `use` is stable anyway. Evolu really should use Suspense instead.

**Comment by steida** [comment]
This API does not exist anymore.
